app/components/versions-graph.js

```diff
@@ -0,0 +1,105 @@
+import semver from 'semver';
+import Component from '@ember/component';
+
+// Component to plot downloadeds split by semver minor version.
+export default Component.extend({
+  resizeHandler: undefined,
```

Comment:
Why is this `undefined`?

Reply:
I was using `download-graph.js` as inspiration and that's what happens there. We can initialise `resizeHandler` at this location and it seems to work ok. Which do you prefer?